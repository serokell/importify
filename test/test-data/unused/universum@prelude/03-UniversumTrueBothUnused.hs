module UniversumTrueBothUnused where

import           Prelude   (Bool (True))
import           Universum hiding (Bool (True), show)

foo :: Bool
foo = undefined
