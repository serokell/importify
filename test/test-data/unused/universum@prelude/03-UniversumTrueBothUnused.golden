module UniversumTrueBothUnused where

import           Prelude   (Bool       )
import           Universum hiding (Bool             )

foo :: Bool
foo = undefined
